ui/app/AppLayouts/Communities/helpers/PermissionsHelpers.qml

```diff
@@ -36,6 +36,13 @@ QtObject {
         return ""
     }
 
+    function getTokenDecimalsByKey(model, key) {
+        const item = getTokenByKey(model, key)
+        if (item)
+            return item.decimals ?? "0"
```

Comment:
Is the decimals property type a string type?

Reply:
fixed